packages/appkit-new/src/utils/TypesUtil.ts

```diff
@@ -72,9 +73,10 @@ export type AppKitOptions = {
    * @see https://cloud.walletconnect.com/
    */
   metadata?: Metadata
+  /**
+   * UniversalProvider instance to be used by AppKit.
+   * AppKit will generate its own instance by default in none provided
+   * @default undefined
+   */
+  walletConnectProvider?: IUniversalProvider
```

Comment:
Should we call it `walletConnectProvider` (for company name matter) or `universalProvider` (or similar)?